invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -360,6 +360,7 @@ <h4>{{ _("Request access") }}</h4>
 {%- endblock page_body %}
 
 {%- block javascript %}
+  <script type="text/javascript" src="{{ config.THEME_MATHJAX_CDN }}"></script>
```

Comment:
```suggestion
  {% if config.THEME_MATHJAX_CDN %}
  <script type="text/javascript" src="{{ config.THEME_MATHJAX_CDN }}"></script>
  {% endif %}
```

As suggested above.

Reply:
Now that I think about it, shouldn't this bee already defined like this in `invenio-theme`? Where did the `THEME_` prefix come from?

Reply:
There's  mathjax defined in `invenio-theme`. Took this constant name from the legacy https://github.com/zenodo/zenodo/blob/master/zenodo/config.py#L1427